test/SConscript

```diff
@@ -13,7 +13,11 @@ test_env.AppendUnique(CCFLAGS=lcov_flags,
                       LIBS='boost_unit_test_framework')
 
 program = test_env.Program(target='test',
-                           source=[env.Glob('*.cpp')])
+                           source=['async_connect_test.cpp',
+                                   'main.cpp',
+                                   'blocking_connect_test.cpp',
+                                   'connector_test.cpp',
+                                   'auth_info_test.cpp'])
```

Comment:
So we are not exercising the MariaDB test here? I'm OK with another follow-up PR to fix MariaDB part, though.

Reply:
Because the MariaDB implement needs mariadb_client and boost.beast.1.68, and it seems there's no easy way to make it run in CI. I had made some [tricks](https://github.com/hhggit/amy/commit/b46b81fed166b7a78b792bb032c82406a5a81ce8) to make it work at my branch [hhggit/mariadb-ci ](https://travis-ci.org/hhggit/amy/jobs/426563525), but I don't think it good enough to merge.